ros2/naive/src/node.cpp

```diff
@@ -0,0 +1,19 @@
+#include "naive/incrementer.hpp"
+
+#include "rclcpp/rclcpp.hpp"
+
+int main(int argc, char **argv) {
+
```

Comment:
Extra newlines

Reply:
Is there like a reference or style guide for new lines in code?